### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zeibura**
- reorganised the readme file for better readability - converted bullet points into sections

- grouped all autocomplete functionalities under one header

- a few points compressed into fewer words or clarified

- included info about bitlang being in the stacks+code4z in the stuff about third party syntax coloring plugins. We will be linking to this readme file from the foundation site, brightside doc and from the code4z readme so I think it's a good idea.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,51 +15,45 @@ COBOL Language Support standardizes the communication between language tooling f
...
 	
+### Copybook Support
+To use the COBOL Language Support extension while working with copybooks, place the copybook files inside a **"COPYBOOKS"** folder within the current workspace. The extension includes the following copybook support features:
+
+* Semantic analysis for keywords, variables, and paragraphs across copybooks, to ensure and maintain compatibility of copybooks called in code.
+* Inbuilt protection against recursive and missing copybooks. If the copybook is missing or contains looping code, an error displays, preventing issues only being discovered when the code is executed.  
+* Variables and paragraphs are defined across copybooks. This ensures consistency of code, and prevents issues in error diagnostics caused by incorrect variables or paragraphs in code. 
+* Functionality to skip variable levels when called, reducing call time.
+* Find All References and Go To Definition functionalities.
+** **Find All References** identifies all occurrences of variables and paragraphs from copybooks in the code.
```
Comment (fox0r) [inline README.md]: I think u wanted to indent a sub items here - usually that's done by just SPACE and *

Comment (zeibura) [reply README.md]: Oops, sorry, fixed, forgot this was MD and not adoc.

**Comment by zeibura** [comment]
Everything in the lead is linked up now, including all extensions (links to VS Code)

**Comment by temanbrcom** [comment]
This GIF shows the old behavior, without specific icons. I think it is better to create new ones.

**Comment by zeibura** [comment]
Dan has new gifs and will add them here

**Comment by DStatWriter** [comment]
CLSAutocorrect
CLSErrorHighlighting

Added gif direct in dev branch, apologies, but here are the files for reference

**Comment by temanbrcom** [comment]
Please, squash the commits before the merge

File: clients/cobol-lsp-vscode-extension/README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 [![GitHub issues](https://img.shields.io/github/issues-raw/eclipse/che-che4z-lsp-for-cobol)](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues)
 [![slack](https://img.shields.io/badge/chat-on%20Slack-blue)](https://join.slack.com/t/che4z/shared_invite/enQtNzk0MzA4NDMzOTIwLWIzMjEwMjJlOGMxNmMyNzQ1NWZlMzkxNmQ3M2VkYWNjMmE0MGQ0MjIyZmY3MTdhZThkZDg3NGNhY2FmZTEwNzQ)
 <a href="https://sonarcloud.io/dashboard?id=eclipse_che-che4z-lsp-for-cobol">
-<img src="https://sonarcloud.io/images/project_badges/sonarcloud-black.svg" width="94" height="20" href="" />
```
Comment (temanbrcom) [inline clients/cobol-lsp-vscode-extension/README.md]: Why do we need this replacement?

**Pull request merged**